parcels/field.py

```diff
@@ -74,6 +74,8 @@ class Field(object):
     :param time_periodic: To loop periodically over the time component of the Field. It is set to either False or the length of the period (either float in seconds or datetime.timedelta object).
            The last value of the time series can be provided (which is the same as the initial one) or not (Default: False)
            This flag overrides the allow_time_interpolation and sets it to False
+    :param name_maps (opt.): gives a name map to the NetCDFFileBuffer that declared a mapping between chunksize name, NetCDF dimension and Parcels dimension;
+           required only if unknown OCM field is loaded and chunking is used by 'field_chunksize' (which is the default)
```

Comment:
'unkown' is a bit strange here. Suggests 'unknown' to the user, but you mean within parcels. Better use 'non-compatible' or something?

Reply:
agreed.